### PR TITLE
Support running multiple bash commands and compile in one query

### PR DESCRIPTION
**Pull request by DonggeLiu**

### CODE REVIEW

**Comment by DonggeLiu** [comment]
Reports:
https://llm-exp.oss-fuzz.com/Result-reports/scheduled/2024-11-30-weekly-all-2/
https://llm-exp.oss-fuzz.com/Result-reports/scheduled/2024-12-02-weekly-all-3/

**Review by mihaimaruseac — APPROVED** [review]

**Review by oliverchang — APPROVED** [review]

File: agent/base_agent.py [hunk 1]
```diff
@@ -63,6 +62,11 @@ def _parse_tag(self, response: str, tag: str) -> str:
     match = re.search(rf'<{tag}>(.*?)</{tag}>', response, re.DOTALL)
     return match.group(1).strip() if match else ''
 
+  def _parse_tags(self, response: str, tag: str) -> list[str]:
+    """Parses the XML-style tags from LLM response."""
+    matches = re.findall(rf'<{tag}>(.*?)</{tag}>', response, re.DOTALL)
```
[the remark this answers is not in the document]
Comment (DonggeLiu) [reply agent/base_agent.py:67]: Done: https://github.com/google/oss-fuzz-gen/issues/743

**Comment by DonggeLiu** [comment]
https://llm-exp.oss-fuzz.com/Result-reports/ofg-pr/2024-12-06-736-dg-comparison/index.html

**Review by mihaimaruseac — APPROVED** [review]

**Comment by DonggeLiu** [comment]
The report looks good: 28/38
https://llm-exp.oss-fuzz.com/Result-reports/ofg-pr/2024-12-13-736-dg-comparison/index.html

I will remove the `libdwarf` benchmark set, which was used for testing only.

**Pull request merged**